docs/components/organization/organization-list.mdx

```diff
@@ -266,19 +266,14 @@ The `<OrganizationList />` component accepts the following properties, all of wh
   ```
 </If>
 
-## Force organizations
+## Personal workspaces disabled (organization‑first)
 
-If you would like to prohibit people from using their personal accounts and force them to be part of an organization, the `hidePersonal` property forces your user to join or create an organization in order to continue. For more information on how to hide personal accounts and force organizations, see the [dedicated guide](/docs/organizations/force-organizations).
+Personal workspaces are disabled by default. When they're disabled, every user **must** belong to an organization.
```

Comment:
This is true only for newest applications. I'm concerned existing app owners will come to our docs and think this is the behavior for them...

I think we should add a callout here.